files/lovely-icons/update.json

```diff
@@ -3,6 +3,6 @@
   "name": "lovely-icons",
   "repo": "douglasdeodato/lovely-icons",
   "files": {
-    "include": ["./css/lovely-icons.min.css", "./lovely-icons/*"]
+    "include": ["./css/lovely-icons.min.css", "./lovely-icons/lovely-icons.min.css", "./lovely-icons/*"]
```

Comment:
That's probably why the bot hasn't picked up 0.1.2. If it doesn't exist anymore, you should remove it from update.json.